hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -667,8 +667,7 @@ public static long addEpochToTxId(long epoch, long txId) {
    */
   @VisibleForTesting
   public static long getTxIdFromObjectId(long objectId) {
-    return ((Long.MAX_VALUE >> REVERSE_EPOCH_ID_SHIFT) & objectId)
-        >> TRANSACTION_ID_SHIFT;
+    return (0x3FFFFFFFFFFFFFFFL & objectId) >> TRANSACTION_ID_SHIFT;
```

Comment:
Can we revert this change? Let's not change any of the non-test code. 

Btw, this does not seem to be used anymore, it seemed it was use in previous non-ratis tests. We can remove this in the future.